src/components/TodoForm.tsx

```diff
@@ -0,0 +1,49 @@
+
+import React from "react"
+
+function TodoForm(props: {
+        todos: any[]; onAdd: (arg0: {
+            // add 1 to current largest id
+            id: number; text: string; done: boolean;
+        }) => void;
+    }) {
+    const [formData, setFormData] = React.useState(
+      {
+        // add 1 to current largest id
+        id: Math.max(...props.todos.map(val => val.id)) + 1,
+        text: "",
+        done: false
+      }
+    );
+
+    function changeHandler(e: { target: { name: any; value: any; }; }) {
```

Comment:
Why `any`?

Reply:
It was suggested by VS Code, I was in a rush to complete the app, so I just clicked the equivalent of "ok, let's do it like you say".